docs/source/markdown/podman-stats.1.md.in

```diff
@@ -34,19 +34,19 @@ Valid placeholders for the Go template are listed below:
 |---------------------|--------------------------------------------------|
 | .AvgCPU             | Average CPU, full precision float                |
 | .AVGCPU             | Average CPU, formatted as a percent              |
-| .BlockInput         | Block Input                                      |
-| .BlockIO            | Block IO                                         |
-| .BlockOutput        | Block Output                                     |
+| .BlockInput         | Total data read from block device                |
+| .BlockIO            | Total data read/total data written to block device|
```

Comment:
This row's last `|` is not in `sync` with other row, I mean this expands table horizontally. Just wondering if this is rendering fine.

Reply:
It renders fine, and I dont think it is worth pushing.